certified-operators/mongodb-enterprise/mongodb-enterprise.v0.10.0.clusterserviceversion.yaml

```diff
@@ -3,20 +3,69 @@
 apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
-  name: mongodboperator.v0.3.2
+  name: mongodboperator.v0.10.0
```

Comment:
By modifying this CSV directly, users with currently running mongodb operators won't be able to update with OLM. The preferred way to introduce a new version of an operator is to create a new CSV with the changes, with the `spec.replaces` field set to the name of the previous CSV for the operator.